capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="number" id="organizationID" name="organizationID"> 
+        <br><br>
```

Comment:
The user probably don't have to input their own id. If you need the id to be a field in the input form you can set the type to be "hidden" and then preset the value.

Reply:
Also University can be hidden too right? we can get that from the user

Reply:
+1 for `type="hidden"`